sdk/samples/iot/provisioning/src/paho_iot_provisioning_example.c

```diff
@@ -471,4 +513,4 @@ static void print_az_span(az_span span)
 
   putchar('\n');
   return;
-}
+}
```

Comment:
Nit: missing NL at EOF.

Reply:
and it caused the macOS build to fail!  fascinating :)